lilit.sargsyan/repo/Lesson_3/exercise_01/exercise_01.py

```diff
@@ -0,0 +1,16 @@
+print('*' *12 + '\n' + '*'*12 + '\n'+ '*'*12 + '\n' + '*'*12)
+
+line_1_lesson_3= "*"*12 +'\n'
```

Comment:
LW: Do not assign

Reply:
Does that mean the solution of  requirement of the problem a and b are the same?